leasing/report/report_base.py

```diff
@@ -332,6 +268,90 @@ def data_as_excel(  # NOQA C901 'ReportBase.data_as_excel' is too complex
+
+            field_format_name = self.get_output_field_attr(field_name, "format")
+
+            match field_format_name:
+                case "date":
+                    field_format = formats[FormatType.DATE]
+                case "percentage":
+                    field_format = formats[FormatType.PERCENTAGE]
+                    if field_value:
+                        field_value /= 100
+                case "money":
+                    if field_value != 0:
+                        field_format = formats[FormatType.MONEY]
+                case "boolean":
+                    if field_value:
+                        field_value = str(_("Yes"))
+                    else:
+                        field_value = str(_("No"))
+                case "area":
+                    field_format = formats[FormatType.AREA]
+
+            field_serializer_field = self.get_output_field_attr(
+                field_name, "serializer_field"
+            )
+            if isinstance(field_serializer_field, ChoiceField):
+                field_value = (
+                    str(field_serializer_field.choices.get(field_value))
+                    if field_value
+                    else ""
+                )
+
+            worksheet.write(row_num, column, field_value, field_format)
+            column += 1
+
+    def write_input_field_value_rows(
+        self, worksheet, report_form, row_num_cursor, formats
+    ):
+        """
+        Writes form input values from report form onto given worksheet at given row number. Requires formats as well.
+        Returns new row num at end of row data written by function.
+        """
+        for input_field_name, input_field in report_form.fields.items():
+            worksheet.write(
+                row_num_cursor,
+                0,
+                "{}:".format(input_field.label),
+                formats[FormatType.BOLD],
+            )
+            field_format = None
+            if input_field.__class__.__name__ == "DateField":
+                field_format = formats[FormatType.DATE]
+
+            input_value = report_form.cleaned_data[input_field_name]
+            if hasattr(input_field, "choices"):
+                for choice_value, choice_label in input_field.choices:
+                    if choice_value == input_value:
+                        input_value = str(choice_label)
+                        break
+
+            if isinstance(input_value, Model):
+                input_value = str(input_value)
+
+            if isinstance(input_value, bool):
+                if input_value:
+                    input_value = gettext("Yes")
+                else:
+                    input_value = gettext("No")
+
+            if isinstance(input_value, QuerySet):
+                input_value = ", ".join([str(v) for v in input_value.all()])
+
+            worksheet.write(row_num_cursor, 1, input_value, field_format)
+            row_num_cursor += 1
+
+        row_num_cursor += 1
```

Comment:
Is this last increment before return intended? It's not present in the original code location.

Reply:
Good spot! The extra empty row is already provided on lines 252 and 273. Removed the extra trailing row.